src/Fable.Core/Fable.Core.Types.fs

```diff
@@ -15,19 +19,34 @@ type CaseRules =
     /// FooBar -> foobar
     | LowerAll = 5
 
-/// Used on interfaces to mangle member names. This allows overloading and prevents conflicts
+/// <summary>
+/// Used on interfaces to mangle member names. This allows overloading and prevents conflicts with
 /// other interfaces, but will make interop with native code more difficult.
+/// </summary>
 type MangleAttribute(mangle: bool) =
     inherit Attribute()
     new() = MangleAttribute(true)
 
-/// Used on a class to attach all members, useful when you want to use the class from JS.
+/// <summary>
+/// Used on a class to attach all members without mangling.
+/// </summary>
+/// <remarks>
+/// If you want to have all members attached to a class (as is standard with most target languages)
+/// and not-mangled, use the <c>AttachMembers</c> attribute. But be aware that overloads won't work
```

Comment:
This feels like a repeat of the `summary`


I am not confortable with this sentence:

> (as is standard with most target languages)

```fs
/// <summary>
/// Used on a class to attach all members without mangling, be aware that overloads won't work
/// </summary>
```